crates/oxc_linter/src/config/config_builder.rs

```diff
@@ -675,4 +711,170 @@ mod test {
             }
         }
     }
+
+    #[test]
+    fn test_extends_rules_single() {
+        let base_config =
+            config_store_from_path("../../apps/oxlint/fixtures/extends_config/rules_config.json");
```

Comment:
NIT: I do not like that we are referencing files from another crate.

Reply:
yeah I don't like that either. I've opened a PR on the top of the stack to fix this later: https://github.com/oxc-project/oxc/pull/9513